debian/postrm

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -e
+
+
+case "$1" in
+    purge|remove)
+        if [ -d "/var/lib/ubuntu-advantage" ]; then
+            rm -rf /var/lib/ubuntu-advantage
+        fi
+
+        # On Trusty clean up the ESM sources.list
+        if grep -iq trusty /etc/os-release; then
+            if [ -f "/etc/apt/sources.list.d/ubuntu-esm-trusty.list" ]; then
+                rm /etc/apt/sources.list.d/ubuntu-esm-trusty.list
```

Comment:
This one I think is fine on purge *and* remove, because, without the apt hook (which is gone when the package is removed), the sources.list file is useless. Would like a second opinion, though.

Reply:
+1